micro-ui/web/micro-ui-internals/packages/modules/pgr/src/pages/citizen/Create/index.js

```
@@ -0,0 +1,130 @@
+    if (nextStep === null) {
+      wrapperSubmit();
+    } else {
+      history.push(`${match.path}/${nextStep}`);
+    }
+  }, [params, nextStep]);
+
+  const goNext = () => {
+    const currentPath = pathname.split("/").pop();
+
+    let { nextStep } = config.routes[currentPath];
+    let compType = Digit.SessionStorage.get(PGR_CITIZEN_CREATE_COMPLAINT);
```

Biome

[error] This let declares a variable that is only assigned once.

'compType' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)